app/containers/ActionSheet/ActionSheet.tsx

```diff
@@ -132,11 +132,12 @@ const ActionSheet = React.memo(
 		const renderItem = ({ item }: any) => <Item item={item} hide={hide} theme={theme} />;
 
 		const animatedPosition = React.useRef(new Value(0));
-		const opacity = interpolate(animatedPosition.current, {
+		// TODO: Similar to https://github.com/wcandillon/react-native-redash/issues/307#issuecomment-827442320
+		const opacity = interpolateNode(animatedPosition.current, {
```

Comment:
https://docs.swmansion.com/react-native-reanimated/docs/2.2.0/migration#1-interpolate-renamed-to-interpolatenode